sfcasts/parameters.md

```diff
@@ -7,15 +7,15 @@ bin/console debug:container
 ```
 
 But these services aren't the only things in our container. It also holds *parameters*. Run the same command as before, but this time, add
-`--parameters`
+the `--parameters` option:
 
 ```terminal-silent
 bin/console debug:container --parameters
 ```
 
 These are basically just variables that you can reference in your code. Most of these are internal, but there *are* some core parameters that you may find useful. For example, any of these that start with `kernel.`, like `kernel.environment`, which is set to the `APP_ENV` environment variable. *Or* this `kernel.project_dir` that's set to the path to the project root directory.
 
-*So* how do we get this from our container? We actually have a special shortcut method for it in our controller. In the `/src` directory, open `Controller/MainController.php`. In `homepage()` method, right on the beginning, say `dd($this->getParameter())`. And inside *that*, say the parameter name wrapped in `%%`: `'%kernel.project_dir%'`. As you can see Symfony has already autocompleted that for us. *Nice*.
+*So* how do we get this from our container? We actually have a special shortcut method for it in our controller. In the `/src` directory, open `Controller/MainController.php`. In the `homepage()` method, right at the beginning, write `dd($this->getParameter())`. And inside *that*, write the parameter name wrapped in `%%`: `'%kernel.project_dir%'`. As you can see, PhpStorm (with the Symfony plugin), has already autocompleted that for us. *Nice*.
```

Comment:
@bocharsky-bw, can you confirm this change?

Reply:
Absolutely correct 👍